### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/update/update-interactive.html

### PR DESCRIPTION
**Pull request by verma-kunal**
This PR is a part of ongoing `hi` localization effort here #29353

/language hi

### CODE REVIEW

File: content/hi/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <div class="katacoda__alert">
+                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट मोड का उपयोग करें
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/6" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button">&lt; मॉड्यूल 5 पर जाए<span class=""></span></a>
+               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">कुबेरनेट्स मूल बातें पर लौटें<span class=""></span></a>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/update/update-interactive.html:30]:
```suggestion
               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button">&lt; मॉड्यूल 5 पर वापस जाएं<span class=""></span></a>
               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">मुलभुत कुबेरनेट्स पर लौटें<span class=""></span></a>
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Final Review.
Do commit these changes if you agree and we should be all set to merge this PR.

**Comment by verma-kunal** [comment]
any other changes to be made here?

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @verma-kunal

**Pull request merged**